core/plugin/zplugin.go

```diff
@@ -22,6 +22,7 @@ import (
 	_ "github.com/coredns/coredns/plugin/erratic"
 	_ "github.com/coredns/coredns/plugin/errors"
 	_ "github.com/coredns/coredns/plugin/etcd"
+	_ "github.com/coredns/coredns/plugin/fanout"
```

Comment:
This needs to be added to `plugin.cfg`. zplugin.go is generated.

Reply:
thanks, fixed.